### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jraddaoui**
Same as with the other PR, any feedback is welcome and will be added to the other repos.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,233 @@
...
+[Install] documentation for more information to install these requirements.
+
+Additionally, follow the [Manage Docker as a non-root user] post-install guide
+so that you don’t have to run Tilt with `sudo`. _Note that managing Docker as a
+non-root user is **different** from running the docker daemon as a non-root user
+(rootless)._
+
+### Requirements for development
+
+While we run the services inside a Kubernetes cluster we recomend to install
```
Comment (djjuhasz) [inline README.md:82]: Edit: "we recommend ~to install~installing Go and other tools locally" (emphasis

File: README.md [hunk 2]
```diff
@@ -1,3 +1,233 @@
...
+
+Bring up the environment:
+
+```bash
+tilt up
+```
+
+While the Docker images are built/downloaded and the Kubernetes resources are
+created, hit `space` to open the Tilt UI in your browser. Check the [Tilt UI]
+documentation to know more about it.
```
Comment (djjuhasz) [inline README.md:124]: Edit: "Check the [Tilt UI] documentation to ~know~learn more about ~it~Tilt.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,233 @@
...
+
+#### TRIGGER_MODE_AUTO
+
+Enables live updates on code changes for the preprocessing worker.
+
+### Tilt UI helpers
+
+#### Submit
+
+In the Tilt UI header there is a cloud icon/button that allows to trigger the
```
Comment (djjuhasz) [inline README.md:192]: Edit: "...a cloud icon/button that ~allows to~can trigger the preprocessing workflow."

**Review by djjuhasz — APPROVED** [review]
Looks good @jraddaoui, just a few minor grammar notes.

**Comment by jraddaoui** [comment]
Thanks @djjuhasz!

**Pull request merged**